egs/mgb2/ASR/conformer_ctc/convert_transcript_words_to_tokens.py

```diff
@@ -0,0 +1 @@
+../../../librispeech/ASR/conformer_ctc/convert_transcript_words_to_tokens.py
```

Comment:
For instance, this symlink is not correct.

Please fix other symlinks reported by the CI.